libraries/module-minifier/src/index.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+/// <reference types="node" preserve="true" />
```

Comment:
Why is this necessary? I see this in a few other packages as well

Reply:
It's a behavior change in TS 5.5